numpy/_build_utils/tempita/_tempita.py

```diff
@@ -784,7 +784,7 @@ def parse_expr(tokens, name, context=()):
     expr = expr.strip()
     if expr.startswith("py:"):
         expr = expr[3:].lstrip(" \t")
-        if expr.startswith("\n") or expr.startswith("\r"):
+        if expr.startswith(("\n", "\r")):
```

Comment:
I will just ignore it, but this is vendor'd code and style fixes seem odd.  When you run these tools, do you use `.gitattributes` to ignore vendor'd code?  (and should we add tempita there if it is helpful for such tools at least.)

Reply:
As far as I know, ruff does not read `.gitattributes`. Is marking vendored files in `.gitattributes` standard practice? Which tools use it?

I can revert changes on vendored code. On the other hand, tempita is unmaintained as far as I know, it doesn't make much sense to struggle to keep consistence with upstream.

Reply:
yeah, dunno if it is worth it.  Github uses the info, I am not sure what other tools do.